orchestra/migrations/0044_auto_20160426_0044.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-04-26 00:44
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('orchestra', '0043_tasktimer_description'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
```

Comment:
The missing migration file was caught, we can add another check, is `AlterModelOptions` something we want to be manually reviewed?